schema.json

```diff
@@ -0,0 +1,214 @@
+          "type": "string",
+        },
+        "name": {
+          "description": "Path to source file under test",
+          "type": "string",
+        },
+        "covered_percent": {
+          "$ref": "#/definitions/covered_percent"
+        },
+        "covered_strength": {
+          "$ref": "#/definitions/covered_strength"
+        },
+        "line_counts": {
+          "$ref": "#/definitions/line_counts"
+        }
+      },
+      "required": ["name", "blob_id", "coverage"],
+    },
+
+    "environment": {
+      "description": "Information and context related to the environment where tests are run",
+      "type": "object",
+      "properties": {
+        "gem_version": {
+          "description": "Simplecov gem version",
+          "type": "string",
+          "pattern": "^\d{1,}\.\d{1,}\.\d{1,}"
+        },
+        "package_version": {
+          "description": "Version of test coverage library in use",
+          "type": "string",
+        },
+        "pwd": {
+          "description": "Working directory where test reporter is executed from",
+          "type": "string",
+          "pattern": "^/"
+        },
+        "rails_root": {
+          "description": "Root of Rails repository as reported by Rails",
+          "type": "string",
+        },
+        "reporter_version": {
+          "description": "Version of Code Climate test reporter tool used to send coverage",
+          "type": "string",
+        },
+        "simplecov_root": {
+          "description": "Simplecov root as reported by Simplecov",
+          "type": "string",
+        },
+      },
+    },
+
+    "git": {
+      "description": "Information about the state of the git repository where coverage is being reported from",
+      "type": "object",
+      "properties": {
+        "branch": {
+          "description": "Current git branch",
+          "type": "string"
+        },
+        "committed_at": {
+          "description": "UNIX timestamp when HEAD committed",
+          "type": "integer",
+        },
+        "head": {
+          "description": "Commit sha of HEAD",
+          "$ref": "#/definitions/git_sha"
+        }
+      },
+      "required": ["branch", "committed_at", "head"],
+    },
+    "ci_service": {
+      "description": "Build related data as reported by CI service which ran tests",
+      "type": "object",
+      "properties": {
+        "branch": {
+          "description": "Branch of code under test",
+          "type": "string",
+        },
+        "build_identifier": {
+          "description": "Unique id of build",
+          "type": "string",
+        },
+        "build_url": {
+          "description": "URL to build on CI service",
+          "type": "string",
+        },
+        "commit_sha": {
+          "description": "Git commit sha checked out for test",
+          "$ref": "#/definitions/git_sha"
+        },
+        "committed_at": {
+          "description": "UNIX timestamp when HEAD committed",
```

Comment:
Do you think it's worth having `#/definitions/timestamp` for this and the other UNIX-timestamp fields?